experimental/packages/opentelemetry-instrumentation-http/test/functionals/http-enable.test.ts

```diff
@@ -167,7 +173,12 @@ describe('HttpInstrumentation', () => {
 
       it('should generate valid spans (client side and server side)', async () => {
         const result = await httpRequest.get(
-          `${protocol}://${hostname}:${serverPort}${pathname}`
+          `${protocol}://${hostname}:${serverPort}${pathname}`,
+          {
+            headers: {
+              'user-agent': 'tester'
+            }
+          }
```

Comment:
Feels a bit weird that we need to update this previously passing test because of the change in the PR.
Can you explain the motivation?

Reply:
It is used to check if the throwing hook is suppressed, configured above in the before hook.